woodwork/column_accessor.py

```diff
@@ -30,6 +30,17 @@
 ks = import_or_none('databricks.koalas')
 
 
+def _check_series_schema(func):
```

Comment:
thoughts on adding this and the dataframe wrapper function to `accessor_utils.py`?